src/ontology/config/icd10cm_exclusion_reasons.robot.template.tsv

```diff
@@ -0,0 +1,173 @@
+term_id	exclusion_reason
+ID	AI rdfs:seeAlso
+ICD10CM:B95-B97	disease causing agents
```

Comment:
There should use only _coded_ exclusion reasons, like we do for ORDO. (`MONDO:excludeRare`). Can you work with @sabrinatoro to make sure the correct codes are used here, not the "labels" of the exclusion reasons?

Reply:
@joeflack4 We have this: https://mondo.readthedocs.io/en/latest/editors-guide/exclusion-reasons/ (also [yaml here](https://github.com/monarch-initiative/mondo/blob/master/src/schema/mondo.yaml). Is this what you are asking?

Reply:
@sabrinatoro I think that .yml will do it!